docs/guidesdraft/01-getting-started.md

```diff
@@ -0,0 +1,228 @@
+## Getting started
+
+### Introduction
+
+Welcome to the CARTO VL guides! This documentation is meant to lead you from the basics of creating a map to advanced techniques for developing powerful interactive visualizations.
+
+In this guide, you will learn how to visualize information with CARTO VL by displaying a dataset on the top of a world basemap. If there is any word or term you do not understand while reading the guides, please take a look to our [Glossary](https://carto.com/developers/carto-vl/guides/glossary).
+
```

Comment:
I know that `https://carto.com/developers/carto-vl/guides/glossary` is not a valid link right now, but we will review all the links on the fly.

Reply:
@elenatorro could we link to specific terms in that glossary? Like https://carto.com/developers/carto-vl/guides/glossary#GIS

That would be very useful!

Reply:
Totally! There is an open issue for the glossary https://github.com/CartoDB/carto-vl/issues/979, we will do that for sure :)